### PR TITLE
atomvm: update to 0.6.1

### PR DESCRIPTION
**Pull request by pguyot**
###### Tested on

macOS 14.4.1 23E224 x86_64
Xcode 15.3 15E204a

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: erlang/atomvm/Portfile [hunk 1]
```diff
@@ -9,8 +9,8 @@ PortGroup               legacysupport 1.1
 # clock_gettime
 legacysupport.newest_darwin_requires_legacy 16
 
-github.setup            atomvm atomvm 0.6.0 v
-revision                1
+github.setup            atomvm atomvm 0.6.1 v
+revision                0
 epoch                   1
 categories              erlang devel
 maintainers             {pguyot @pguyot} openmaintainer
```
Comment (reneeotten) [inline erlang/atomvm/Portfile:16]: your e-mail address is not correct, they way you have it now implies a ...@macports.org address. You should add the (obfuscated) domain as shown in the guide.

**Review by reneeotten — COMMENTED** [review]
thanks for the update @pguyot ! One small detail I just noticed, other than that this looks good to me!

File: erlang/atomvm/Portfile (hunk 1 above)
Comment (pguyot) [reply erlang/atomvm/Portfile:16]: @reneeotten Thank you for your review. I really mean to use my macports.org address for this, and I checked, it still works.

Comment (reneeotten) [reply erlang/atomvm/Portfile:16]: oh, interesting... sorry, I see now that you indeed have a ...@macports.org e-mail address. You're listed though under retired developers/maintainers on the Wiki, that's probably why the bot doesn't label your PR as "Member" and why you don't have commit access.

**Pull request merged**